microsoftgraph/microsoft-graph-explorer-v4
# Fix: Encode plus sign

## Overview
Fixes #1470 

### Demo
![image](https://user-images.githubusercontent.com/45680252/156405676-0345f1bc-c7c8-4b48-815f-5b4611b95a1a.png)
Unauthenticated case

![image](https://user-images.githubusercontent.com/45680252/156406021-198ab8d4-fe9f-46c3-bfc7-6b391cd9904f.png)
Authenticated case

Notes
The endpoint does not recognize plus signs. All plus signs are converted to spaces as seen in the postman call below
![image](https://user-images.githubusercontent.com/45680252/156500973-b6c30a2f-b54a-4c01-9975-850a21168425.png)

Encoding the plus signs before making the API call works
![image](https://user-images.githubusercontent.com/45680252/156501143-c255c152-16a1-4ba2-a4ee-417eec7de067.png)
Notice that the plus sign (+) has been encoded as %2b

This is therefore not an SDK issue, but a service issue


## Testing Instructions

* How to test this PR
* Start after checking out this branch
* 
- Copythe query: https://graph.microsoft.com/v1.0/me/calendarView?startDateTime=2020-01-01T19:00:00+05:30&endDateTime=2022-01-01T19:00:00+05:30 on the query input field
- Run the query
- Notice that the query runs successfully
